File: AAA_Sample_Story_Bazar/Bazar.js
function main() {
    var elephant = 10;
    var bicycles = 10;
    var sunflower = 10;
    const initialPrice = elephant + bicycles + sunflower;
    elephant += 3;
    sunflower -= 5;
    bicycles = elephant + 2;
    const finalPrice = elephant + bicycles + sunflower;

    const gainOrLoss = finalPrice - initialPrice;
    console.log(`in the hend the bazar seller has gained ${gainOrLoss} $`);
    if (gainOrLoss < 0) {
        console.log('so his idea to change prices was not smart');
    } else {
        console.log('he was a smart dude and could fill his pouch');
    }
}

main();

File: AAA_Sample_Story_Bazar/Bazar.dart
main() {
  int elephant = 10;
  int bicycles = 10;
  int sunflower = 10;
  int initialPrice = elephant + bicycles + sunflower;
  elephant += 3;
  sunflower -= 5;
  bicycles = elephant + 2;
  int finalPrice = elephant + bicycles + sunflower;

  int gainOrLoss = finalPrice - initialPrice;
  print('in the end the bazar seller has gained $gainOrLoss \$');
  print('so his idea to change prices was smart');
}
